### PR TITLE
Update samples

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/search/azure-search-documents/samples/async_samples/sample_vector_search_async.py [hunk 1]
```diff
@@ -152,7 +154,9 @@ async def single_vector_search_with_filter():
     async with search_client:
         results = await search_client.search(
             search_text="",
-            vector=Vector(value=get_embeddings(query), k=3, fields="descriptionVector"),
```
Comment (pvaneck) [inline sdk/search/azure-search-documents/samples/async_samples/sample_vector_search_async.py:155]: Seems we can remove the Vector import at the top now

File: sdk/search/azure-search-documents/samples/sample_vector_search.py [hunk 2]
```diff
@@ -133,7 +133,9 @@ def single_vector_search():
 
     results = search_client.search(
         search_text="",
-        vector=Vector(value=get_embeddings(query), k=3, fields="descriptionVector"),
```
Comment (pvaneck) [inline sdk/search/azure-search-documents/samples/sample_vector_search.py:136]: Same here, can remove the Vector import as well as DAC import

**Review by pvaneck — APPROVED** [review]
Some cleanup comments, but lgtm

**Comment by farzad528** [comment]
lgtm

**Pull request merged**